spec/koans.js

```diff
@@ -3,65 +3,77 @@ context = describe;
 describe("`let` vs. `var`.", () => {
   it("the scope for `var` is not restricted to the block", () => {
     if (true) {
-      /* use the correct keyword to initialize `varX` here */
+      // use the correct keyword to initialize `varX` here 
+      let varX;
```

Comment:
here the idea was to use the `var` keyword and show how it leaks from the if statement